### PR TITLE
Fix code style violation of rule MEN002

### PR DESCRIPTION
**Pull request by v-pegao**
Workitem is [here](https://dev.azure.com/ceapex/Engineering/_workitems/edit/107202?src=WorkItemMention&src-action=artifact_link)

|Rule|Fix|
|---|---|
|MEN002: Lines too long<br/>(current line limit is 140 characters)|- If it is caused by too many nested brackets, try  turn local functions into member functions or use early return<br/>- If it is long parameter, try put parameters to a new line, if it is still too long, break each parameter into its own line<br/>- If parameter is a long expression, try extract a variable<br/>- For long member variable declarations with initialization, try put the default initializer to a new line<br/>- If the return value is a tuple ant the tuple definition is long, try put the return value definition in its own line<br/>- Suppress this warning in Errors.cs|

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/5020)

### CODE REVIEW

File: src/docfx/build/Build.cs [hunk 1]
```diff
@@ -38,17 +38,25 @@ public static async Task Run(string docsetPath, CommandLineOptions options, Erro
                 (Docset docset, Docset fallbackDocset) GetDocsetWithFallback()
```
Comment (yufeih) [inline src/docfx/build/Build.cs]: try turn inner function into member function first

Comment (v-pegao) [reply src/docfx/build/Build.cs]: Turned inner function `GetDocsetWithFallback()` and `ShouldBuildFile()` to member function, and resolved the line too long issue.

File: src/docfx/Errors.cs [hunk 2]
```diff
@@ -3,10 +3,12 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 
 namespace Microsoft.Docs.Build
 {
+    [SuppressMessage("Layout", "MEN002", Justification = "Suppress MEN002 for Errors.cs")]
```
Comment (yufeih) [inline src/docfx/Errors.cs]: 👍 want to help replace places where we use `#pragma warning disable` with Attribute?

Comment (v-pegao) [reply src/docfx/Errors.cs]: Sure. Will update in other PR later.

**Pull request merged**